build.sbt

```diff
@@ -47,7 +47,8 @@ lazy val WeaponRegeX = projectMatrix
     name := "weapon-regex",
     libraryDependencies += "com.lihaoyi" %%% "fastparse" % "2.3.3",
     libraryDependencies += "org.scalameta" %%% "munit" % "0.7.29" % Test,
-    tpolecatScalacOptions += ScalacOption("-Xsource:3", _.major == 2)
+    tpolecatScalacOptions += ScalacOption("-Xsource:3", _.major == 2),
+    tpolecatScalacOptions += ScalacOption("-Xlint:unused", _.major == 2)
```

Comment:
This option is already enabled, I think: 

https://docs.scala-lang.org/overviews/compiler-options/index.html

![image](https://user-images.githubusercontent.com/10114577/198826039-4cc5b2a9-0642-4421-9c3a-34afc7573653.png)

![image](https://user-images.githubusercontent.com/10114577/198826086-c8410c56-1677-439d-a4d8-d351c7c76730.png)

Reply:
Strange 🤔 I got [this error](https://github.com/stryker-mutator/weapon-regex/actions/runs/3350979650/jobs/5552131281) `@nowarn annotation does not suppress any warnings`, which seems like what is described [here](https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html#:~:text=To%20ensure%20that%20%40nowarn%20annotations%20actually%20suppress%20warnings%2C%20enable%20%2DXlint%3Aunused%20or%20%2DWunused%3Anowarn.%20With%20this%20option%2C%20the%20compiler%20checks%20that%20every%20%40nowarn%20annotation%20suppresses%20at%20least%20one%20warning%20and%20issues%20a%20warning%20otherwise%3A).

Reply:
Stryker4s disables some compiler options to prevent unrelated compiler errors, that might be causing it

Reply:
Then how should I resolve this?